### ISSUE TITLE
Inspector:  Audio playback is playing only once

### ISSUE THREAD

**Issue by golddotasksquestions** [godotengine/godot#36816]
**Godot version:** 3.2.stable

**OS/device including version:**Win64

**Issue description:** 
Audio playback in the Inspector is playing only once. User has to press stop or double click the file in the Filebrowser to be able to play again.

![playback_audiofile](https://user-images.githubusercontent.com/47016402/75983094-2090b700-5ee8-11ea-9c2e-15d116496379.gif)

Instead I would expect if the playback finished, simply clicking on the playbutton should suffice to start playback again.

**Steps to reproduce:** 

1. double click an audiofile in the Filebrowser
2. go to the Inspector and click the playbutton 
3. after the playback finished, click the play button again.

**Comment by bruce965** [comment]
I can confirm this bug on c9768f15f (v4.0.dev.custom_build) on Linux Mint 19.3.
Only *.wav files show this issue.

**Comment by bruce965** [comment]
I opened a pull request to fix this issue, but I also noticed a strange behaviour in the `AudioStreamEditor`.

When the "stop" button is pressed, the cursor seeks back to the start of the stream. But when "play" is pressed again, the playback doesn't resume from the start; it resumes from where the stream was playing when "stop" was pressed last time.

I would assume this is not the intended behaviour and I can open another pull request to change it if necessary.

Thanks.

**Comment by golddotasksquestions** [comment]
@bruce965 I do think this is intended behavior, because if you press play, the icon changes to a pause icon, indicating that if you press it again _while_ it is in playback, it will pause, but not stop the playback.

**Comment by bruce965** [comment]
@golddotasksquestions I agree that the "pause" button should preserve the position, but what about the "stop" button? What's the difference between "pause" and "stop"?

**Comment by golddotasksquestions** [comment]
Pressing a stop button stops the playback and resets the position immediately to the beginning.
Pressing a pause button pauses the playback at the position it is currently at until

- pause button is pressed again (resuming playback)
- play button is pressed (in cases where play/pause share the same button: resuming playback, in case of separate buttons: play/pause button, starting playback from the beginning)
- stop button is pressed (reset timestamp to the beginning)

Unfortunately this universal principle that exists since decades for any kind of video, audio or animation playback has not at all been consistently adopted in Godot. See:
https://github.com/godotengine/godot-proposals/issues/167
https://github.com/godotengine/godot-proposals/issues/287

**Comment by madprogramer** [comment]
I can also confirm the error to be present on the MacOS version, I was going to open a new issue but instead decided to comment here.

**Comment by KoBeWi** [comment]
This seems to be fixed in 3.2.4 rc3
![156GVmzwmp](https://user-images.githubusercontent.com/2223172/109574231-dd175b80-7aef-11eb-80b2-4d3100e8cbf4.gif)
Can someone else confirm?

**Comment by golddotasksquestions** [comment]
Just checked, it is still present for me in Godot 3.2.4 rc3, Win64

**Comment by KoBeWi** [comment]
It's wav and I'm on Windows 10. Maybe I'll try other files.
